### PR TITLE
[13.0][MIG] stock_barcodes_automatic_entry

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Standard migration

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

File: stock_barcodes_automatic_entry/__manifest__.py [hunk 1]
```diff
@@ -8,12 +8,12 @@
         with the class 'barcode-automatic-entry' after a barcode scanned has
         been processed.
     """,
-    "version": "11.0.1.0.0",
+    "version": "13.0.1.0.0",
     "license": "AGPL-3",
     "maintainers": ["ageficent"],
     "author": "Eficent Business and IT Consulting Services S.L.,"
```
Comment (AdriaGForgeFlow) [inline stock_barcodes_automatic_entry/__manifest__.py]: Can you update this and the copyrights as well?

**Review by AdriaGForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Travis still complaining :S

File: stock_barcodes_automatic_entry/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "Stock Barcodes Automatic Entry",
+    "summary": """
+        This module will automatically trigger the click event on a button
+        with the class 'barcode-automatic-entry' after a barcode scanned has
+        been processed.
+    """,
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "maintainers": ["ageficent"],
+    "author": "ForgeFlow S.L.,"
```
Comment (rousseldenis) [inline stock_barcodes_automatic_entry/__manifest__.py]: missing one space after comma

Comment (AaronHForgeFlow) [reply stock_barcodes_automatic_entry/__manifest__.py]: :+1:

**Review by JoanSForgeFlow — APPROVED** [review]
Functional review, tested on local. :+1:

**Pull request merged**